org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/actions/IJavaEditorActionDefinitionIds.java

```diff
@@ -394,6 +394,14 @@ public interface IJavaEditorActionDefinitionIds extends ITextEditorActionDefinit
 	 */
 	String EXTRACT_CLASS= "org.eclipse.jdt.ui.edit.text.java.extract.class"; //$NON-NLS-1$
 
+	/**
+	 * Action definition ID of the refactor -> make static action
+	 * (value <code>"org.eclipse.jdt.ui.edit.text.java.make.static"</code>).
+	 *
+	 * @since 3.31
```

Comment:
This public api change will require bumping jdt.ui to 3.32.0 and this since tag will need changing to 3.32.